Assets/Mirror/Runtime/Transport/LLAPITransport.cs

```diff
@@ -110,8 +110,14 @@ public override bool ClientConnected()
             return clientConnectionId != -1;
         }
 
-        public override void ClientConnect(string address)
+        public override void ClientConnect(Uri uri)
         {
+            if (uri.Scheme != "llapi")
```

Comment:
why is this scheme "llapi" and not "udp", where as telepathy's scheme is "tcp" and not "telepathy"?

Reply:
We can pick anything we want.   I don't feel strongly about "llapi", "udp", "tcp",  or any other schemes.   let's discuss in discord what is appropriate?

Currently we have the following transports:
* Telepathy
* Apathy
* Booster
* Websocket  ("ws" and "wss" are the natural picks for this)
* Steam
* litenetlib
* Ignorance

Telepathy and Apathy are wire compatible,  so they could potentially use the same scheme.  All others are not compatible with each other,  so in my mind it makes sense to use different schemes for them.